image/templates/helm/shared/internal/scanner-v4-config-shape.yaml

```diff
@@ -0,0 +1,81 @@
+scannerV4:
+  disable: null # bool
+  indexer:
+    metricsPort: null # int
```

Comment:
perhaps `indexer` and `matcher` could have a `db` field to configure the connection to the db such as
```
indexer:
  db:
    connectionString:
      secretName: ...
    tls:
      secretName: ...
matcher:
  db:
    connectionString:
      secretName: ...
    tls:
      secretName: ...
```

Reply:
I don't expect anything to change between indexer and matcher, but this does allow for the flexibility. If it's not annoying to update to, then I'm ok with it. If it's a huge pain, then let's just keep it as-is to get this merged asap